src/components/BackgroundImage/index.test.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+
+import { render, screen } from '@testing-library/react';
+
+import BackgroundImage from '.';
+
+describe('BackgroundImage', () => {
+  const dataTestId = 'background-image';
+  it('renders BackgroundImage with the background image', () => {
```

Comment:
Please add a linebreak between block 🙏 
```suggestion
  const dataTestId = 'background-image';

  it('renders BackgroundImage with the background image', () => {
```